src/integration/kotlin/pl/edu/agh/gem/integration/ability/CurrencyManagerAbility.kt

```diff
@@ -45,7 +44,7 @@ fun stubCurrencyManagerExchangeRate(body: Any?, baseCurrency: String, targetCurr
                     .withStatus(statusCode.value())
                     .withAppContentType()
                     .withBody(
-                        jacksonObjectMapper().writeValueAsString(body),
+                        jacksonObjectMapper().registerModules(JavaTimeModule()).writeValueAsString(body),
```

Comment:
if you want you could by default register that module in config

Reply:
tried to add ```jacksonObjectMapper().registerModules(JavaTimeModule())``` ApplicationConfig in init function but it looks like that jacksonObjectMapper() always return new Object and registering only works if we use the same object for registering and writing. Considered creating a bean but it would require making stub functions (that use jacksonObjectMapper) part of class that has been injectet with jacksonObjectMapper bean